bakery/src/tasks/assemble-book-metadata.js

```diff
@@ -33,7 +33,13 @@ const task = (taskArgs) => {
           book_dir="assembled-book/$collection_id"
           target_dir="assembled-book-metadata/$collection_id"
           mkdir "$target_dir"
-          assemble-meta "$book_dir" "$target_dir/collection.assembled-metadata.json"
+
+          echo "{" > uuid-to-revised-map.json
+          find assembled-book/$collection_id/raw/ -path */m*/metadata.json | xargs cat | jq -r '. | "\"\(.id)\": \"\(.revised)\","' >> uuid-to-revised-map.json
+          echo '"dummy": "dummy"' >> uuid-to-revised-map.json
```

Comment:
Is there a reason we're adding these `dummy` entries for `uuid-to-revised-map.json` and `module-canonicals.json` in various task scripts?

Reply:
JSON does not allow trailing commas, so I have to include and extra entry without one. I was hoping there was a better way to create valid json, but this was the only thing I came up with in a reasonable amount of time. Know of a way that might be more neat?

Reply:
Ah, I see. Nothing obvious comes to mind in terms of alternatives. 🤷‍♂️